cmd/builder/internal/command.go

```diff
@@ -24,11 +24,21 @@ import (
 	"github.com/knadh/koanf/providers/file"
 	"github.com/knadh/koanf/providers/posflag"
 	"github.com/spf13/cobra"
+	flag "github.com/spf13/pflag"
 	"go.uber.org/zap"
 
 	"go.opentelemetry.io/collector/cmd/builder/internal/builder"
 )
 
+const SkipComplicationFlag = "skip-compilation"
```

Comment:
I personally find it easier to read when this is grouped, like:

```golang

const (
    SkipComplicationFlag = ...
    DistributionNameFlag = ...
)
```

Note that we should skip the compilation, not the complication. We have to remain complicated!

Reply:
Lol I'll make the change

Reply:
Done, although I stand by SkipComplication 😄